README.md

```diff
@@ -29,7 +29,7 @@ Project | Aras
 
 ### Pre-requisites
 
-1. Aras Innovator installed (version 11.0 SPx preferred)
+1. Aras Innovator installed (current released version preferred)
```

Comment:
Let's make this a little more generic. The "current released version" might not be preferred or even supported, especially as releases update over time and projects don't keep up.
```suggestion
1. Aras Innovator installed (version ___ preferred)
```